From 399639e561460fa2e18621e98d57983f86102a21 Mon Sep 17 00:00:00 2001
From: Richard Nelson <wabz@pidgin.im>
Date: Sun, 13 May 2007 00:32:47 +0000
Subject: [PATCH] gntworkspace doc

--HG--
branch : finch.workspaces
extra : close : 1
extra : convert_revision : 76cfc9565a5cdde8d2e1f2efc3282b309a55a110
---
 doc/finch.1.in |   36 ++++++++++++++++++++++++++++++++++++
 1 file changed, 36 insertions(+)

diff --git a/doc/finch.1.in b/doc/finch.1.in
index 483a813..c625792 100644
--- a/doc/finch.1.in
+++ b/doc/finch.1.in
@@ -109,6 +109,15 @@ Jump to the 1st, 2nd ... 10th window.
 .B Ctrl \+ o
 Bring up the menu (if there is one) for a window. Note that currently only the
 buddylist has a menu.
+.TP
+.B Alt \+ Shift \+ .
+Switch to the next workspace
+.TP
+.B Alt \+ Shift \+ ,
+Switch to the previous workspace
+.TP
+.B Alt \+ s
+Show the workspace list
 
 .SH FILES
 \fI~/.gntrc\fR: configuration file for gnt applications.
@@ -134,6 +143,33 @@ remember_position = 1
 .br
 
 .br
+# Workspaces are created simply by adding Workspace-X groups as follows:
+.br
+[Workspace-1]
+.br
+name = blist
+.br
+# window-names specifies that windows with these semi-colon separated names are placed
+into this workspace
+.br
+window-names = buddylist;debug-window
+.br
+
+.br
+[Workspace-2]
+.br
+name = IM
+.br
+window-names = conversation-window
+.br
+# window-titles specifies that windows with these semi-colon separated titles are placed
+into this workspace. These are matched as substrings. Window titles take precedence over
+names.
+.br
+window-titles = Preferences;Pounce
+.br
+
+.br
 [colors]
 .br
 # The RGB values range in [0, 1000]
-- 
1.7.10.4

